src/modules/extendedCallWindow/assets/selectedVehicleCounter.ts

```diff
@@ -45,6 +45,7 @@ export default (LSSM: Vue, MODULE_ID: string, btnVehicles: string[]): void => {
     table.style.setProperty('border-radius', '4px');
     table.style.setProperty('overflow', 'auto');
     table.style.setProperty('width', 'auto');
+    table.style.setProperty('color', '#333');
```

Comment:
Thanks for your contribution!
Unfortunately, your change makes the text unreadable in darkmode:
![grafik](https://user-images.githubusercontent.com/27238105/211852753-f33b69f4-9782-46a9-b9e1-9f41d45b99fc.png) ![grafik](https://user-images.githubusercontent.com/27238105/211852822-e960e46a-901d-4321-909b-2206c0e53dec.png)

Fix:
```suggestion
    if (!LSSM.$stores.root.isDarkMode)
        table.style.setProperty('color', '#333');
```

Reply:
Changed as requested